### ISSUE TITLE
React error with v7.1.0

### ISSUE THREAD

**Issue by slavanga** [KittyGiraudel/react-a11y-dialog#284]
After updating to v7.1.0 i’m getting the following React error:
> Error: Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in, or you might have mixed up default and named imports.

Using `A11yDialog` with `next` and here is a minimal example to reproduce the issue:
https://stackblitz.com/edit/stackblitz-starters-xkqmqb?file=pages%2Findex.js

**Comment by KittyGiraudel** [comment]
I can reproduce with a fresh Next app:

```
npx create-next-app test-rad
npm install react-a11y-dialog
# Import & use it in index.js
npm run dev
```

So far I don’t understand why.

**Comment by KittyGiraudel** [comment]
@mxmason If you have an idea, I’m all ears. I guess the library is not compiled properly with Rollup, but I don’t understand what’s wrong. :(

**Comment by jgarplind** [comment]
Ran into the same issue I'm afraid.

For me, it only appears when I run unit tests.

When running my app on localhost, I instead get this warning (which I did not get before 7.1.0):

> Warning: Failed prop type: The prop `classNames.closeButton` is marked as required in `A11yDialog2`, but its value is `undefined`.

Hope it might help spark some ideas as to what has changed.

**Comment by KittyGiraudel** [comment]
What changed is that we use Rollup instead of Babel to compile the library since v7, but I have no clue what the actual problem is (besides an incorrect Rollup config). 😅

**Comment by mayank99** [comment]
would you consider removing prop-types? it's a legacy feature and not really necessary when using typescript with functional components.

**Comment by KittyGiraudel** [comment]
@mayank99 We could! I’m just not sure that would help?

**Comment by mayank99** [comment]
ah you're right, the props-type issue is different and not present in the latest.

so i went to inspect the package and found that it bundles a bunch of transitive dependencies from react, which is likely not desired.

i tested a build with all node modules marked external, and it compiles fine. (there is still an issue with prop-types, but it might be because of the default value not including a required property). see https://stackblitz.com/edit/stackblitz-starters-wyzabm?file=pages%2Findex.js,react-a11y-dialog.js

**Comment by KittyGiraudel** [comment]
I believe it’s should be fixed in [7.1.3](https://github.com/KittyGiraudel/react-a11y-dialog/releases/tag/7.1.3). I would love a confirmation. :)

**Comment by slavanga** [comment]
I can confirm that the issue is resolved on my end 🤩
Thank you @mayank99 and @KittyGiraudel!
